spec/fortnox/api/mappers/unit_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+require 'fortnox/api'
+require 'fortnox/api/mappers/unit'
+require 'fortnox/api/models/unit'
+require 'fortnox/api/mappers/examples/mapper'
+
+module Fortnox
+  module API
+    # Shhh Rubocop, we don't need a comment here ... Really
+    module Mapper
+      describe Unit do
+        describe 'when mapping model' do
+          let(:model) { Model::Unit.new(code: 'lbs', description: 'Pounds') }
```

Comment:
I would change `Model::Unit` to `described_class::Model::Unit` and remove the module declarations in this spec.